articles/container-registry/container-registry-oci-artifacts.md

```diff
@@ -51,6 +41,52 @@ az acr login --name myregistry
 > [!NOTE]
 > `az acr login` uses the Docker client to set an Azure Active Directory token in the `docker.config` file. The Docker client must be installed and running to complete the individual authentication flow.
 
+### Sign in with ORAS
+
+This section shows options to sign into the registry. Choose one method below appropriate for your environment.
+
+Run  `oras login` to authenticate with the registry. You may pass [registry credentials](container-registry-authentication.md) appropriate for your scenario, such as service principal credentials, user identity, or a repository-scoped token (preview).
+
+- Authenticate with your [individual Azure AD identity](container-registry-authentication.md?tabs=azure-cli#individual-login-with-azure-ad) to use an AD token.
+
+  ```azurecli
+  USER_NAME="00000000-0000-0000-0000-000000000000"
```

Comment:
Do we need to say:
Always use "000..." as the token is parsed through the `PASSWORD` variable.